ClassevivaPCTO/Dialogs/NoticeDialogContent.xaml

```diff
@@ -49,10 +49,10 @@
 
                     <AppBarButton x:Name="ButtonOpen" Label="{CustomResource NoticeDialogOpenAttachmentButton}"
                                   Click="ButtonOpen_Click" Icon="OpenFile"
-                                  VerticalAlignment="Center" />
+                                  VerticalAlignment="Center" ToolTipService.ToolTip="Apri file nel browser"/>
```

Comment:
mettere in risorse

Reply:
ci stanno un bordello di strings da mettere in risorse, volevo fare una pr apparte